### DISCUSSION TITLE
Defined badges

### DISCUSSION THREAD

**Discussion by vanillajonathan** [GradienceTeam/Gradience#672] [Ideas]
It seems that badges exist but that they are not widely used yet.

For the badges to be meaningful, I would like to propose that all badges must be from a predefined set.

I would like to propose a set of badges:
- Light
- Dark
- Low contrast
- High contrast
- [WCAG](https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines)
- Without blue (blue light suppresses the body's release of melatonin)

**Comment by daudix** [comment]
Not sure about blue but yeah, sounds good (we actually have concept for those badges [here](https://github.com/GradienceTeam/Design/blob/main/Palettes/colors.png))

**Reply by vanillajonathan** [reply]
I wasn't aware that you had a concept for badges already. It is unclear to me what the "Favorite" badge is supposed to indicate.

From within Gradiance it just lists all themes and the badge seemingly is only visible once you have installed it, and then expanded the entry in the list of installed themes.

**Reply by daudix** [reply]
There is other concept

https://github.com/GradienceTeam/Design/blob/main/Mockups/Gradience%20-%20Preset%20Manager%20V2.png

**Reply by vanillajonathan** [reply]
Yes, this looks much more useful with badges visible in the Explore tab.
